packages/open-next/src/build/helper.ts

```diff
@@ -254,3 +254,20 @@ export function copyOpenNextConfig(
     path.join(outputPath, "open-next.config.mjs"),
   );
 }
+
+export function copyEnvFile(
+  appPath: string,
+  packagePath: string,
+  outputPath: string,
+) {
+  const baseAppPath = path.join(appPath, ".next/standalone", packagePath);
+  const baseOutputPath = path.join(outputPath, packagePath);
+  const envPath = path.join(baseAppPath, ".env");
+  if (fs.existsSync(envPath)) {
+    fs.copyFileSync(envPath, path.join(baseOutputPath, ".env"));
+  }
+  const envProdPath = path.join(baseAppPath, ".env.production");
```

Comment:
would it makes sense to support `.env.staging` or `.env.<arbitary-staging-env>`?

Reply:
I don't think we should, next don't copy `.env.local` , `.env.development` or `.env.staging` into the standalone output.
Maybe there is some other, but i think we cover much cases here